content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -40,30 +40,40 @@ It's good practice to structure your files well, so we'll expect you to have a s
+
+Later in this program you'll be learning how to train machine learning models using various kinds of data. When doing so, missing values cannot simply be ignored. Further, when it comes to datasets with many columns (also known as features), it is bad practice to drop an entire row for a few missing values. 
+
+Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+
 
 ## Questions
 
 The [data set](data.csv) consists of health and demographic data for the period 2014-2015, obtained from [Global Health Observatory Data Repository](http://apps.who.int/gho/data/node.main). Here is some [metadata](data-info.txt) that may be useful.
 
-1. Are there any missing values in the data set? You can deal with missing values by either dropping them or filling them with some value (in a variety of ways) or ignoring them. What do you think is best in this case? And why? Here is a resource for dealing with missing values : https://www.analyticsvidhya.com/blog/2021/10/a-complete-guide-to-dealing-with-missing-values-in-python/
+1. Load the dataset into a pandas DataFrame called `health_and_demographics_df`,  display the first 10 rows and determine how many missing values there are per feature. 
+
+2. Address any missing values in the dataset. Given the kind of data we have in this dataset, as well as the *regional* analysis you will be tasked with performing below, it should not simply be replacing a missing value with the mean/median of the whole column. 
 
-2. Are there any other problems with the data? If so, fix them.
+3. Are there any other problems with the data? If so, fix them. At this stage ensure that your dataset is in a DataFrame named: `health_and_demographics_df_cleaned`
 
-3. Identify the country with the lowest % of their population under 15 and the one with the highest.
+4. Identify the country with the lowest % of their population under 15 and the one with the highest and save each country name as a string in the variables `country_with_lowest_population_percentage_under_15` and `country_with_highest_population_percentage_under_15` respectively.
 
-4. Which region has the highest % of their population over 60?
+5. Which region has the highest % of their population over 60? Save this region in the the variable `region_with_highest_population_percentage_over_60`
 
-5. Does fertility decrease as income increases? Are there any countries that don't seem to follow this relation?
+6. Does fertility decrease as income increases? Are there any countries that don't seem to follow this relation?
```

Comment:
Is this a rhetorical question? Or is the learner expected to do write down or store the answer somewhere?